index.html

```diff
@@ -5,70 +5,79 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>surveyform</title>
-		<link rel="stylesheet" href="./assets/styles/style.css">
+<link rel="stylesheet" href="./assets/styles/style.css">
 </head>
 <body>
-	<section id="background">
-		<header class="topic">
-			<h1>freeCodeCamp Survey Form</h1>
-			<p>thank you for taking the time to help us improve the platform</p>
-		</header>
-
-		<!--personal info-->
-
-		<section id="informations">
-				<div class="question">
-					<p>Name</p>
-					<input class="name" type="text" placeholder="Enter your name" required>
-				</div>
-				<div class="question">
-					<p>Email</p>
-					<input class="name" type="text" placeholder="Enter your email" required>
-				</div>
-				<div class="question">
-					<p>Age (optional)</p>
-					<input class="name" type="text" placeholder="Enter your age">
-				</div>
-				<div class="question">
-					<p>Which option best describe you current role?</p>
-					<input class="name" type="" placeholder="Enter your name" required>
-				</div>
-			
-
-			<!--personal info-->
-
-			<section id="true">
-				<form action="/submit">
-					<p>Would you recommend freeCodeCamp to a friend?</p>
-					<lable><input type="radio" name="Definitely-Maybe-Not sure">Definitely</lable><br>
-					<lable><input type="radio" name="Definitely-Maybe-Not sure">Maybe</lable><br>
-					<lable><input type="radio" name="Definitely-Maybe-Not sure">Not sure</lable><br>
-				</form>
-			</section>	
-
-			<!--feature-->
-
-			<section>
-				<p>What is your favorite feature of freeCodeCamp?</p>
-				<input class="name" type="" placeholder="Select an option"><br>
-				<p>What would you like to see improved? (Check all that apply)</p>
-				<label><input type="checkbox">Font-end project</label><br>
-				<label><input type="checkbox">Back-end projet</label><br>
-				<label><input type="checkbox">Data visualisation</label><br>
-				<label><input type="checkbox">Challenges</label><br>
-				<label><input type="checkbox">Open source community</label><br>
-				<label><input type="checkbox">Gitter helps rooms</label><br>
-				<label><input type="checkbox">Videos</label><br>
-				<label><input type="checkbox">City Meetup</label><br>
-				<label><input type="checkbox">Wiki</label><br>
-				<label><input type="checkbox">Forums</label><br>
-				<label><input type="checkbox">Additional courses</label><br>
-			</section>
-
-			<!--comment-->
-		</section>
-
-	</section>
-
```

Comment:
This project does not require page sectioning you just have a background image and a form that contains the information needed. Please make sure you follow that pattern.